HACKING.adoc

```diff
@@ -229,6 +229,74 @@ opam switch create my-switch-name --empty
 opam pin add ocaml-variants.$VERSION+branch git+https://$REPO#branch
 ----
 
+=== Incremental builds with `opam`
+
+If you want to fix a bug that needs opam packages to reproduce, or want to try
+a new feature that you're still actively developing on opam packages, you will
+likely need a faster feedback loop than re-creating a new switch or recompiling
+every package each time you make a small change to your code.
```

Comment:
This feels a bit redundant and/or should be integrated with the intro of the preceding section. Then at that point I would keep only something like "In order to get a faster feedback loop there are mostly two orthogonal way:"